ansible/install_glance.yaml

```diff
@@ -16,46 +16,31 @@
       path: "{{ glance_yaml_dir }}"
       state: directory
 
-  - name: Copy deployment yamls to working dir
+  - name: Copy files to yaml dir
     copy:
       src: "{{ item }}"
       dest: "{{ glance_yaml_dir }}/"
-    loop:
-    - "ocp/glance/1"
-    - "ocp/glance/2"
-    - "ocp/glance/3"
+    with_fileglob:
+    - "ocp/glance/*"
 
-  - name: Create database
-    shell: |
-      set -e
-
-      oc apply -f "{{ glance_yaml_dir }}/1/"
-      oc wait -n openstack job/glance-dbcreate \
-          --for condition=Complete --timeout=60s
-    environment:
-      PATH: "{{ oc_env_path }}"
-      KUBECONFIG: "{{ kubeconfig }}"
-
-  - name: Create db schema
+  - name: Start glance
     shell: |
       set -e
-
-      oc apply -f "{{ glance_yaml_dir }}/2/"
-      oc wait -n openstack job/glance-dbsync \
-          --for condition=Complete --timeout=60s
-      oc wait -n openstack job/glance-service-init \
-          --for condition=Complete --timeout=60s
+      oc apply -n openstack -f "{{ glance_yaml_dir }}/secret.yaml"
+      oc apply -n openstack -f "{{ glance_yaml_dir }}/glance.yaml"
+      oc apply -n openstack -f "{{ glance_yaml_dir }}/service-init-cmds.yaml"
+      oc apply -n openstack -f "{{ glance_yaml_dir }}/service-init-job.yaml"
     environment:
       PATH: "{{ oc_env_path }}"
       KUBECONFIG: "{{ kubeconfig }}"
 
-  - name: Start glance
+  - name: wait for deployment to be created
     shell: |
-      set -e
-
-      oc apply -f "{{ glance_yaml_dir }}/3/"
-      oc wait -n openstack deployment/glance \
-          --for condition=available --timeout=60s
```

Comment:
We need these, too, after the wait on deployment creation you added. In testing, this is where you'll hit interesting failures if you've deployed but the service isn't starting.

This is important, so I'd prefer to add this back in before merging. Happy to add this as it should only take a minute.